apps/frontend/src/modules/workspace/components/settings/Workspace/components/InviteList.tsx

```diff
@@ -50,55 +50,63 @@ export const InviteList = () => {
     };
 
     return (
-        <div className="mt-6">
+        <div className="mt-6 text-sm">
             {(workspace?.invites?.edges?.length as number) > 0 && (
-                <h3 className="font-semibold">
+                <h3 className="font-semibold text-white">
                     {workspace?.invites?.edges?.length}{" "}
                     {workspace?.invites?.edges && workspace?.invites?.edges?.length > 1 ? "invites" : "invite"}
                 </h3>
             )}
             {workspace?.invites?.edges?.map((invite) => {
                 return (
-                    <div key={invite?.node?.id}>
-                        <div className="flex items-center justify-between px-2 py-3">
-                            <div className="basis-[60%]">
-                                <p className="font-sm font-medium">
-                                    {invite?.node?.firstName} {invite?.node?.firstName}
-                                </p>
-                                <p className="font-sm">{invite?.node?.email}</p>
-                            </div>
-                            <DropdownMenu.Root>
-                                <DropdownMenu.Trigger asChild>
-                                    <button className="w-fit rounded-md px-1 py-1 transition-all duration-300 ease-in hover:cursor-pointer hover:bg-intg-bg-1 data-[state=a]:bg-intg-bg-1">
-                                        <MoreHorizontal color="#AFAAC7" />
-                                    </button>
-                                </DropdownMenu.Trigger>
+                    <div
+                        key={invite?.node?.id}
+                        className="flex items-center justify-between border-b border-intg-bg-4 px-2 py-3"
+                    >
+                        <div className="flex h-[48px] basis-[60%] flex-col justify-center">
+                            <p className="font-sm font-medium capitalize text-white">{`${invite?.node?.firstName}`}</p>
+                            <p className="font-sm">
+                                <span>{invite?.node?.email}</span>
+                                <span>
+                                    {invite?.node?.email === user.email ? (
```

Comment:
@afolasope You can't be invited to an organization if you are already a member of the organization.